src/main/scala/latis/service/hapi/AboutResponse.scala

```diff
@@ -0,0 +1,63 @@
+package latis.service.hapi
+
+import io.circe.Encoder
+
+/**
+ * Response from the `/about` endpoint.
+ *
+ * @param version version of HAPI
+ * @param status HAPI status object
+ * @param id unique ID for the server
+ * @param title human-readable name for the server
+ * @param contact contact info for server issues
+ * @param description brief description of the type of data served
+ * @param contactId ID in the discovery system for info about the contact
+ * @param citation how to cite the server
+ */
+final case class AboutResponse(
+  version: String,
+  status: Status,
+  id: String,
+  title: String,
+  contact: String,
+  description: Option[String] = None,
+  contactId: Option[String] = None,
+  citation: Option[String] = None
+)
+
+object AboutResponse {
+
+  /** JSON encoder */
+  implicit val encoder: Encoder[AboutResponse] = {
+    val f: AboutResponse => Product8[
+      String,
+      Status,
+      String,
+      String,
+      String,
+      Option[String],
+      Option[String],
+      Option[String]
+    ] = ar => (
+          ar.version,
+          ar.status,
+          ar.id,
+          ar.title,
+          ar.contact,
+          ar.description,
+          ar.contactId,
+          ar.citation
+        )
+
+    Encoder.forProduct8(
+      "version",
+      "status",
+      "id",
+      "title",
+      "contact",
+      "description",
+      "contactID",
+      "citation"
+    )(f).mapJson(_.dropNullValues)
```

Comment:
Not super proud of this function `f` that I'm passing to `forProduct8` below, but I can't figure out how to keep it an anonymous function and still play nicely with `.mapJson(_.dropNullValues)`. Scala seems to lose its ability to infer correct types once I tack that onto the end.

Reply:
Looks like you can just add a type to the second argument to `forProduct`:

```scala
Encoder.forProduct8(
    "version",
    "status",
    "id",
    "title",
    "contact",
    "description",
    "contactID",
    "citation"
  ) { (ar: AboutResponse) => (  // <-- Here
    ar.version,
    ar.status,
    ar.id,
    ar.title,
    ar.contact,
    ar.description,
    ar.contactId,
    ar.citation
  ) }.mapJson(_.dropNullValues)
```

Reply:
Omg that's the first thing I tried. But I never put parens around `ar: AboutResponse`, apparently that was the trick.